packages/web-components/fast-foundation/src/accordion/README.md

```diff
@@ -147,6 +147,20 @@ export const myAccordionItem = AccordionItem.compose<AccordionItemOptions>({
 
 <hr/>
 
+### Variables
+
+| Name                  | Description               | Type                                                                                                                                                                                                                                                                                                                             |
+| --------------------- | ------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `AccordionExpandMode` | Expand mode for Accordion | `{
+    /**      * Designates only a single {@link @microsoft/fast-foundation#(AccordionItem:class) } can be open a time.      */
+    single: "single",
+
+    /**      * Designates multiple {@link @microsoft/fast-foundation#(AccordionItem:class) or AccordionItems} can be open simultaneously.      */
+    multi: "multi",
+}` |
```

Comment:
The formatting for these will need to be handled in the `CEMToMarkdown` script

Reply:
I think it should be but definitely odd that it's not. I'll try running everything again locally - not sure why this wouldn't transform.

Reply:
OH - These are the new values coming in. I'll take a look before tagging Bill :)

Reply:
@williamw2 Looks like we may need some custom handling to enable proper docs for moving from enums to const objects - is that something you might be able to help with?

Reply:
I'll take a look.

Reply:
Per @williamw2 I think we'll need some kind of additional formatting solution here. I don't think we should block on this, but we should figure out that solution to resolve the formatting issue.